src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -53,6 +62,130 @@ struct CPU {
         if (_cores == 0) {
             _cores = _processors;
         }
+
+        /**
+         * New method to get CPU infomation and CPU map. Below is the structure of CPU map and two sample.
+         *  1. Four processors of two Pcore
+         *  2. Four processors of four Ecores shared L2 cache
+         *
+         *  Proc ID | Socket ID | HW Core ID | Phy Core of Pcores | Logic Core of Pcores | ID of Ecore Group | Used
+         *     0         1            1                0                      1                   0             0
+         *     1         1            1                1                      0                   0             0
+         *     2         1            2                0                      2                   0             0
+         *     3         1            2                2                      0                   0             0
+         *     4         1            3                0                      0                   1             0
+         *     5         1            4                0                      0                   1             0
+         *     6         1            5                0                      0                   1             0
+         *     7         1            6                0                      0                   1             0
+         */
+        _proc = sysconf(_SC_NPROCESSORS_ONLN);
+        _cpu_mapping.resize(_proc, std::vector<int>(CPU_MAP_USED_PROC + 1, 0));
+
+        auto updateProcMapping = [&](const int nproc) {
+            if (0 == _cpu_mapping[nproc][CPU_MAP_CORE]) {
+                int core_1 = 0;
+                int core_2 = 0;
+                std::string::size_type pos = 0;
+                std::string::size_type endpos = 0;
+                std::string sub_str = "";
+
+                std::ifstream cache_1_file("/sys/devices/system/cpu/cpu" + std::to_string(nproc) +
+                                           "/cache/index0/shared_cpu_list");
+                std::ifstream cache_2_file("/sys/devices/system/cpu/cpu" + std::to_string(nproc) +
```

Comment:
Need confirm other Linux OS(rhel82, centOS, etc) have the same system file name.

Reply:
This is Linux kernel standard path which tested on Ubuntu and Rhel.